lib/Form/Block/Columns.php

```diff
@@ -12,9 +12,12 @@ private function getStyles(array $params): string {
     if (isset($params['text_color'])) {
       $styles[] = "color:{$params['text_color']};";
     }
-    if (isset($params['background_color'])) {
+    if (isset($params['background_color']) && !isset($params['gradient'])) {
```

Comment:
Hmmm, I think this needs an explanation, I don't understand this line of code, can you add a code comment, please?